hikari/api/rest.py

```diff
@@ -2960,6 +2960,12 @@ async def authorize_access_token(
     ) -> applications.OAuth2AuthorizationToken:
         """Authorize an OAuth2 token using the authorize code grant type.
 
+        .. warning::
+            There is no way to ensure what scopes are granted in the token,
```

Comment:
There's seemingly no reason to put this note here since there isn't a `scopes` parameter? Also this change seems outside the scope of this PR